### DISCUSSION TITLE
HLS audio tracks

### DISCUSSION THREAD

**Discussion by Impre-visible** [cookpete/react-player#1783] [Ideas]
#### Current Behavior
I want the React Player, to allow users to select the audio track played, if multiple exists in a hls file.

#### Expected Behavior
HLS.js allow users to select audio tracks, you can test that [on there website](https://hlsjs.video-dev.org/demo/). Being able to do that by default, would be great.

#### Steps to Reproduce
1. Create a m3u8 master file with audio and video tracks
2. Import this m3u8 playlist to ReactPlayer
3. You can't select the track

#### Environment
- URL attempting to play: [url of an m3u8 file]
- Browser: Chrome 123.0.6312.122
- Operating system: Ubuntu 22.04

**Comment by Shoxruxasadov** [comment]
Is the problem still not solved?
